src/detect-byte-extract.c

```diff
@@ -698,18 +423,13 @@ static int DetectByteExtractTest01(void)
 {
     int result = 0;
 
-    DetectByteExtractData *bed = DetectByteExtractParse(NULL, "4, 2, one");
+    SCDetectByteExtractData *bed = DetectByteExtractParse(NULL, "4, 2, one");
```

Comment:
Was not this unit test duplicated in rust ?
Can it go away from C ?

Reply:
There's little benefit, imo, to removing existing test cases. There are rust based unittests but those are not exercised with `suricata -u`

Reply:
I think both rust and C unit tests are run with `make check`

> There's little benefit, imo, to removing existing test cases. 

I would not say it had been removed, just moved ;-)